app/Http/Controllers/LikeController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Meal;
+use App\Models\like;
+
+class LikeController extends Controller
+{
+    public function store(Request $request, $id)
+    {
+        $like = new Like;
+        $meal = Meal::find($id);
+        $like->meal_id = $meal->id;
```

Comment:
$idで取得したMealのidは$idなのでこの処理はそのままidを指定しても同様ですね😅
解答修正しておきます🙇‍♂️

```suggestion
        $like->meal_id = $id;
```